akka-distributed-data/src/main/scala/akka/cluster/ddata/Replicator.scala

```diff
@@ -272,9 +305,9 @@ final class ReplicatorSettings(
       deltaCrdtEnabled,
       200)
 
-  def withRole(role: String): ReplicatorSettings = copy(roles = ReplicatorSettings.roleOption(role).iterator.toSet)
+  def withRole(role: String): ReplicatorSettings = copy(roles = ReplicatorSettings.roleOption(role).toSet)
 
-  def withRole(role: Option[String]): ReplicatorSettings = copy(roles = role.iterator.toSet)
+  def withRole(role: Option[String]): ReplicatorSettings = copy(roles = role.toSet)
```

Comment:
Hmm, why was the iterator there to start with, wasn't that some 2.13 compatibility thing?

Reply:
no clue, maybe something for 2.11